tests/foreman/cli/test_lifecycleenvironment.py

```diff
@@ -411,21 +411,20 @@ def test_environment_paths(self):
         """
         try:
             org = make_org(cached=True)
-            payload = {
+            test_env = make_lifecycle_environment({
                 'organization-id': org['id'],
-            }
-            test_env = make_lifecycle_environment(payload)
+            })
         except CLIFactoryError as err:
             self.fail(err)
 
         # Add paths to lifecycle environments
-        result = LifecycleEnvironment.paths({'organization-id': org['id'],
-                                            'permission-type': 'readable'})
-        self.assertEqual(result.return_code, 0,
-                         "return code must be 0, instead got {0}".
-                         format(result.return_code))
-        self.assertEqual(
-            len(result.stderr), 0,
-            "There should not be an error here.")
-        self.assertIn(u'Library >> {0}'.format(test_env['name']),
-                      result.stdout)
+        result = LifecycleEnvironment.paths({
+            'organization-id': org['id'],
+            'permission-type': 'readable',
+        })
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stderr), 0)
```

Comment:
That output is from nosetests and I have grabbed it from jenkins. Py.test have a similar output:

```
$ py.test tests/foreman/cli/test_computeresource.py -k test_create_positive_libvirt
================================ test session starts ================================
platform darwin -- Python 2.7.6 -- py-1.4.26 -- pytest-2.6.4
collected 26 items

tests/foreman/cli/test_computeresource.py F

===================================== FAILURES ======================================
 TestComputeResource.test_create_positive_libvirt_1__u_name___u__ub34b_u5ed1_ua0a4_ud174_ua295_ua085_u8a31_u71a8_u69fc_ua97c________________________________u_description___u_azdfXnPzQQ_____________________________

self = <tests.foreman.cli.test_computeresource.TestComputeResource testMethod=test_cr..._____________________u_description___u_azdfXnPzQQ_____________________________>

    @wraps(func)
    def wrapper(self):
>       return func(self, *args, **kwargs)

../../.virtualenvs/robottelo/lib/python2.7/site-packages/ddt.py:146:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
tests/foreman/cli/test_computeresource.py:196: in test_create_positive_libvirt
    self.assertEqual(result.return_code, 0)
E   AssertionError: 1 != 0
------------------------------- Captured stdout setup -------------------------------
2015-02-17 18:18:52 - robottelo.common.ssh - DEBUG - >>> [qe-foreman-rhel7.usersys.redhat.com] LANG=en_US.UTF-8 hammer -v -u admin -p changeme  --output csv compute-resource create --name="gFVc9cEQ" --url="qemu+tcp://qe-foreman-rhel7.usersys.redhat.com:16509/system" --provider="Libvirt"
2015-02-17 18:18:52 - robottelo.common.ssh - DEBUG - >>> [qe-foreman-rhel7.usersys.redhat.com] LANG=en_US.UTF-8 hammer -v -u admin -p changeme  --output csv compute-resource create --name="gFVc9cEQ" --url="qemu+tcp://qe-foreman-rhel7.usersys.redhat.com:16509/system" --provider="Libvirt"
2015-02-17 18:18:54 - robottelo.common.ssh - INFO - Instantiated Paramiko client 0x10a0c3f10
2015-02-17 18:18:54 - robottelo.common.ssh - INFO - Instantiated Paramiko client 0x10a0c3f10
2015-02-17 18:18:55 - robottelo.common.ssh - INFO - Destroying Paramiko client 0x10a0c3f10
2015-02-17 18:18:55 - robottelo.common.ssh - INFO - Destroying Paramiko client 0x10a0c3f10
2015-02-17 18:18:55 - robottelo.common.ssh - INFO - Destroyed Paramiko client 0x10a0c3f10
2015-02-17 18:18:55 - robottelo.common.ssh - INFO - Destroyed Paramiko client 0x10a0c3f10
2015-02-17 18:18:55 - robottelo.common.ssh - DEBUG - <<<
Message,Id,Name
Compute resource created,33,gFVc9cEQ
2015-02-17 18:18:55 - robottelo.common.ssh - DEBUG - <<<
Message,Id,Name
Compute resource created,33,gFVc9cEQ
2015-02-17 18:18:55 - robottelo.common.ssh - DEBUG - >>> [qe-foreman-rhel7.usersys.redhat.com] LANG=en_US.UTF-8 hammer -v -u admin -p changeme  compute-resource info --id="33"
2015-02-17 18:18:55 - robottelo.common.ssh - DEBUG - >>> [qe-foreman-rhel7.usersys.redhat.com] LANG=en_US.UTF-8 hammer -v -u admin -p changeme  compute-resource info --id="33"
2015-02-17 18:18:56 - robottelo.common.ssh - INFO - Instantiated Paramiko client 0x10a0cbfd0
2015-02-17 18:18:56 - robottelo.common.ssh - INFO - Instantiated Paramiko client 0x10a0cbfd0
2015-02-17 18:18:58 - robottelo.common.ssh - INFO - Destroying Paramiko client 0x10a0cbfd0
2015-02-17 18:18:58 - robottelo.common.ssh - INFO - Destroying Paramiko client 0x10a0cbfd0
2015-02-17 18:18:58 - robottelo.common.ssh - INFO - Destroyed Paramiko client 0x10a0cbfd0
2015-02-17 18:18:58 - robottelo.common.ssh - INFO - Destroyed Paramiko client 0x10a0cbfd0
2015-02-17 18:18:58 - robottelo.common.ssh - DEBUG - <<<
Id:          33
Name:        gFVc9cEQ
Provider:    Libvirt
Url:         qemu+tcp://qe-foreman-rhel7.usersys.redhat.com:16509/system
Description:
User:
Created at:  2015/02/17 20:18:53
Updated at:  2015/02/17 20:18:53

2015-02-17 18:18:58 - robottelo.common.ssh - DEBUG - <<<
Id:          33
Name:        gFVc9cEQ
Provider:    Libvirt
Url:         qemu+tcp://qe-foreman-rhel7.usersys.redhat.com:16509/system
Description:
User:
Created at:  2015/02/17 20:18:53
Updated at:  2015/02/17 20:18:53

------------------------------- Captured stdout call --------------------------------
2015-02-17 18:19:03 - robottelo - DEBUG - Running test TestComputeResource/test_create_positive_libvirt_1__u_name___u__ub34b_u5ed1_ua0a4_ud174_ua295_ua085_u8a31_u71a8_u69fc_ua97c________________________________u_description___u_azdfXnPzQQ_____________________________
2015-02-17 18:19:03 - robottelo - DEBUG - Running test TestComputeResource/test_create_positive_libvirt_1__u_name___u__ub34b_u5ed1_ua0a4_ud174_ua295_ua085_u8a31_u71a8_u69fc_ua97c________________________________u_description___u_azdfXnPzQQ_____________________________
2015-02-17 18:19:03 - robottelo.common.ssh - DEBUG - >>> [qe-foreman-rhel7.usersys.redhat.com] LANG=en_US.UTF-8 hammer -v -u admin -p changeme  --output csv compute-resource create --url="https://example.org" --description="azdfXnPzQQ[]@#$%^&*(),./?"{}>\<|''" --name="덋廑ꂤ텴ꊕꂅ許熨槼ꥼ[]@#$%^&*(),./?\"{}>\<|''" --provider="Libvirt"
2015-02-17 18:19:03 - robottelo.common.ssh - DEBUG - >>> [qe-foreman-rhel7.usersys.redhat.com] LANG=en_US.UTF-8 hammer -v -u admin -p changeme  --output csv compute-resource create --url="https://example.org" --description="azdfXnPzQQ[]@#$%^&*(),./?"{}>\<|''" --name="덋廑ꂤ텴ꊕꂅ許熨槼ꥼ[]@#$%^&*(),./?\"{}>\<|''" --provider="Libvirt"
2015-02-17 18:19:04 - robottelo.common.ssh - INFO - Instantiated Paramiko client 0x10a0c3e50
2015-02-17 18:19:04 - robottelo.common.ssh - INFO - Instantiated Paramiko client 0x10a0c3e50
2015-02-17 18:19:05 - robottelo.common.ssh - INFO - Destroying Paramiko client 0x10a0c3e50
2015-02-17 18:19:05 - robottelo.common.ssh - INFO - Destroying Paramiko client 0x10a0c3e50
2015-02-17 18:19:05 - robottelo.common.ssh - INFO - Destroyed Paramiko client 0x10a0c3e50
2015-02-17 18:19:05 - robottelo.common.ssh - INFO - Destroyed Paramiko client 0x10a0c3e50
2015-02-17 18:19:05 - robottelo.common.ssh - DEBUG - <<< bash: -c: line 0: syntax error near unexpected token `,./?\"{}'
bash: -c: line 0: `LANG=en_US.UTF-8 hammer -v -u admin -p changeme  --output csv compute-resource create --url="https://example.org" --description="azdfXnPzQQ[]@#$%^&*(),./?"{}>\<|''" --name="덋廑ꂤ텴ꊕꂅ許熨槼ꥼ[]@#$%^&*(),./?\"{}>\<|''" --provider="Libvirt"'

2015-02-17 18:19:05 - robottelo.common.ssh - DEBUG - <<< bash: -c: line 0: syntax error near unexpected token `,./?\"{}'
bash: -c: line 0: `LANG=en_US.UTF-8 hammer -v -u admin -p changeme  --output csv compute-resource create --url="https://example.org" --description="azdfXnPzQQ[]@#$%^&*(),./?"{}>\<|''" --name="덋廑ꂤ텴ꊕꂅ許熨槼ꥼ[]@#$%^&*(),./?\"{}>\<|''" --provider="Libvirt"'

============== 25 tests deselected by '-ktest_create_positive_libvirt' ==============
===================== 1 failed, 25 deselected in 13.12 seconds ======================
```